src/otx/api/entities/shapes/polygon.py

```diff
@@ -28,8 +29,8 @@ class Point:
     __slots__ = ["x", "y"]
 
     def __init__(self, x: float, y: float):
-        self.x = x
```

Comment:
I had some problem with one dataset and that's why I added this. Because of machine precision it was something like 1.00000001  as far as I remember, and it led to errors